baseline-error-prone/src/test/java/com/palantir/baseline/errorprone/MissingCasesInEnumSwitchTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.baseline.errorprone;
+
+import com.google.errorprone.CompilationTestHelper;
+import com.google.errorprone.bugpatterns.MissingCasesInEnumSwitch;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.DisabledForJreRange;
+import org.junit.jupiter.api.condition.JRE;
+
+public class MissingCasesInEnumSwitchTest {
+
+    @Test
+    @DisabledForJreRange(max = JRE.JAVA_13)
+    public void testSwitchExpression() {
```

Comment:
Once the issue is fixed in error-prone this test will begin failing. Unfortunately, we use Java 11 in CI so this test isn't run.

I'll try to remember to come back and revert this after the upstream fix is released.

Reply:
We can add a test which uses the errorprone package implementation version which forces us to take action when the version changes -- that way we're forced to remember :-)